release/release_logs/1.7.0/stress_tests/dead_actors.txt

```diff
@@ -0,0 +1,50 @@
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 561, in ray._raylet.execute_task
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 568, in ray._raylet.execute_task
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 572, in ray._raylet.execute_task
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 522, in ray._raylet.execute_task.function_executor
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/function_manager.py", line 579, in actor_method_executor
+[2m[36m(pid=3700, ip=172.31.49.215)[0m     return method(__ray_actor, *args, **kwargs)
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/util/tracing/tracing_helper.py", line 448, in _resume_span
+[2m[36m(pid=3700, ip=172.31.49.215)[0m     return method(self, *_args, **_kwargs)
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "stress_tests/test_dead_actors.py", line 28, in ping
+[2m[36m(pid=3700, ip=172.31.49.215)[0m SystemExit: -1
+[2m[36m(pid=3700, ip=172.31.49.215)[0m 2021-09-25 09:42:51,916	ERROR worker.py:428 -- SystemExit was raised from the worker
+[2m[36m(pid=3700, ip=172.31.49.215)[0m Traceback (most recent call last):
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 684, in ray._raylet.task_execution_handler
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 524, in ray._raylet.execute_task
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 561, in ray._raylet.execute_task
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 568, in ray._raylet.execute_task
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 572, in ray._raylet.execute_task
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 522, in ray._raylet.execute_task.function_executor
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/function_manager.py", line 579, in actor_method_executor
+[2m[36m(pid=3700, ip=172.31.49.215)[0m     return method(__ray_actor, *args, **kwargs)
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/util/tracing/tracing_helper.py", line 448, in _resume_span
+[2m[36m(pid=3700, ip=172.31.49.215)[0m     return method(self, *_args, **_kwargs)
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "stress_tests/test_dead_actors.py", line 28, in ping
```

Comment:
Do we need these?

Reply:
I think these logs are expected, so we don't need to have them here (just the result is enough)